/**: # wildcard for the node name
  ros__parameters:
    body_name: 'uuv02'
    server_address: '192.168.0.181'
    latch_timeout: false # if true, the node needs to be restarted after mocap timoeut to publish again
    publish_visual_odometry: false # if true, we publish the px4/FCU visual odometry topic
